.github/workflows/neural-networks-library.yaml

```diff
@@ -0,0 +1,29 @@
+name: CI-Neural-Networks-library
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - biblioteca-redes-neuronales
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
```

Comment:
No tengo muy claro tampoco por qué haces esto. No vas a usar matrix, realmente, ni testear para diferentes sistemas operativos. Si adoptáis una plantilla de algún lado, tratad de entender qué hace y si se aplica a tu caso determinado.